R/rjournal_article.R

```diff
@@ -1,35 +1,56 @@
+#' This file is only a basic article template. For full details of _The R
+#' Journal_ style and information on how to prepare your article for submission,
+#' see the [Instructions for Authors](https://journal.r-project.org/share/author-guide.pdf)
 #'
-#' \tabular{lll}{
-#' FIELD \tab TYPE \tab DESCRIPTION\cr
-#' `name` \tab *required* \tab name and surname of the author\cr
-#' `affiliation` \tab *required* \tab name of the author's affiliation\cr
-#' `address` \tab *required* \tab at least one address line for the affiliation\cr
-#' `url` \tab *optional* \tab an additional url for the author or the main affiliation\cr
-#' `orcid` \tab *optional* \tab the authors ORCID if available\cr
-#' `email` \tab *required* \tab the author's e-mail address\cr
-#' `affiliation2` \tab *optional* \tab name of the author's 2nd affiliation\cr
-#' `address2` \tab *optional* \tab address lines belonging to the author's 2nd affiliation
-#' }
+#' ## About this format and the R Journal requirements
+#'
+#' `rticles::rjournal_article` will help you build the correct files requirements:
+#'
+#' - A R file will be generated automatically using `knitr::purl` - see
+#' https://bookdown.org/yihui/rmarkdown-cookbook/purl.html for more information.
+#' - A tex file will be generated from this Rmd file and correctly included in
+#' `RJwapper.tex` as expected to build `RJwrapper.pdf`.
+#' - All figure files will be kept in the default rmarkdown `*_files` folder. This
+#' happens because `keep_tex = TRUE` by default in `rticles::rjournal_article`
+#' - Only the bib filename is to modifed. An example bib file is included in the
+#' template (`RJreferences.bib`) and you will have to name your bib file as the
+#' tex, R, and pdf files.
+#'
+#' # About YAML header fields
+#'
+#' This section documents some of the YAML fields that can be used with this
+#' formats.
+#'
+#'
+#' ## The `author` field in the YAML header
+#'
+#' | FIELD  | TYPE | DESCRIPTION |
+#' | ------ | ---- | ----------- |
+#' | `name` | *required* | name and surname of the author |
+#' | `affiliation` | *required* | name of the author's affiliation |
+#' | `address` | *required* | at least one address line for the affiliation |
+#' | `url` | *optional* | an additional url for the author or the main affiliation |
+#' | `orcid` | *optional* | the authors ORCID if available |
+#' | `email` | *required* | the author's e-mail address |
+#' | `affiliation2` | *optional* | name of the author's 2nd affiliation |
+#' | `address2` | *optional* | address lines belonging to the author's 2nd affiliation |
 #'
 #' *Please note: Only one `url`, `orcid` and `email` can be provided per author.*
 #'
-#' **Other YAML fields**
+#' ## Other YAML fields
+#'
+#' | FIELD | TYPE | DESCRIPTION |
+#' | ----- | ---- | ----------- |
+#' | `bibliography` | *with default* | the BibTeX file with the reference entries |
+#' | `output_dir` | *with default* | output directory where all the files are stored ready for submission |
```

Comment:
The `output_dir` is not anymore part of the YAML-header after the new commit.

Reply:
Good catch ! I am still checking everything - if you see anything else do not hesitate.